engine/src/main/java/org/terasology/world/internal/WorldProviderWrapper.java

```diff
@@ -42,89 +40,54 @@ public WorldProviderWrapper(WorldProviderCore core, ExtraBlockDataManager extraD
         this.extraDataManager = extraDataManager;
     }
 
-    @Override
-    public boolean isBlockRelevant(Vector3i pos) {
-        return core.isBlockRelevant(pos.x, pos.y, pos.z);
-    }
-
     @Override
     public boolean isBlockRelevant(Vector3ic pos) {
         return core.isBlockRelevant(pos.x(), pos.y(), pos.z());
     }
 
-    @Override
-    public boolean isBlockRelevant(Vector3f pos) {
-        return isBlockRelevant(new Vector3i(pos, RoundingMode.HALF_UP));
-    }
-
     @Override
     public boolean isBlockRelevant(Vector3fc pos) {
-        return isBlockRelevant(new org.joml.Vector3i(pos, org.joml.RoundingMode.HALF_UP));
-    }
-
-    @Override
-    public Block setBlock(Vector3i pos, Block type) {
-        return core.setBlock(pos, type);
+        return isBlockRelevant(new Vector3i(pos, RoundingMode.HALF_UP));
```

Comment:
why is it rounding half up?

Reply:
I just took what it was before :thinking: 

Do we have a clear understanding which side is inclusive and which is exclusive?

Reply:
actually its correct I think.